src/main/webapp/app/exercises/text/manage/text-exercise/text-exercise-update.component.html

```diff
@@ -205,7 +205,7 @@ <h4 *ngIf="isImport" id="jhi-text-exercise-heading-import" jhiTranslate="artemis
             <fa-icon [icon]="'ban'"></fa-icon>&nbsp;<span jhiTranslate="entity.action.cancel">Cancel</span>
         </button>
 
-        <button type="submit" (click)="save()" class="btn btn-primary" id="submit-entity">
+        <button type="submit" (click)="save()" class="btn btn-primary" id="submit-entity" [disabled]="editForm.form.invalid || isSaving">
```

Comment:
```suggestion
        <button type="submit" (click)="save()" class="btn btn-primary" id="submit-entity" [disabled]="editForm.form.invalid || isSaving || textExercise.dueDateError || textExercise.assessmentDueDateError">
```
I'm not sure if this is needed but this structure is used for the other save-buttons so I think it should be consistent.

Reply:
Good catch, added the extra checks.